cdk/lib/db.ts

```diff
@@ -308,40 +351,48 @@ export class CacclRdsDb extends CacclDbBase {
       instanceCount = 1,
       instanceType = DEFAULT_DB_INSTANCE_TYPE,
       engineVersion = DEFAULT_AURORA_MYSQL_ENGINE_VERSION,
-      profiler = false,
       databaseName,
     } = props.options;
 
-    const auroraMysqlEngineVersion = RdsDatabaseClusterEngine
-      .auroraMysql({
-        version: AuroraMysqlEngineVersion.of(engineVersion),
-      });
+    const auroraMysqlEngineVersion = rds.DatabaseClusterEngine.auroraMysql({
+      version: rds.AuroraMysqlEngineVersion.of(engineVersion),
+    });
 
     // performance insights for rds mysql not supported on t3 instances
     const enablePerformanceInsights = !instanceType.startsWith('t3');
 
-    this.clusterParameterGroupParams.lower_case_table_names = "1";
-    this.clusterParameterGroupParams.aurora_enable_repl_bin_log_filtering = "1";
+    this.clusterParameterGroupParams.lower_case_table_names = '1';
+    this.clusterParameterGroupParams.aurora_enable_repl_bin_log_filtering = '1';
 
-    const clusterParameterGroup = new RdsParameterGroup(this, 'ClusterParameterGroup', {
-      engine: auroraMysqlEngineVersion,
-      description: `RDS parameter group for ${Stack.of(this).stackName}`,
-      parameters: this.clusterParameterGroupParams,
-    });
+    const clusterParameterGroup = new rds.ParameterGroup(
+      this,
+      'ClusterParameterGroup',
+      {
+        engine: auroraMysqlEngineVersion,
+        description: `RDS parameter group for ${Stack.of(this).stackName}`,
+        parameters: this.clusterParameterGroupParams,
+      },
+    );
 
-    this.instanceParameterGroupParams.slow_query_log = "1";
-    this.instanceParameterGroupParams.log_output = "TABLE";
-    this.instanceParameterGroupParams.long_query_time = "3";
+    this.instanceParameterGroupParams.slow_query_log = '1';
+    this.instanceParameterGroupParams.log_output = 'TABLE';
+    this.instanceParameterGroupParams.long_query_time = '3';
```

Comment:
I'm assuming all params have to be strings then? Just curious about the quotes around numbers